src/App.css

```diff
@@ -1,51 +1,15 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #22201c;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-.brand-title {
-  font-size: 3rem;
-  color: #d9d9d9;
-}
-
-.brand-title span {
-  color: #ffde59;
-}
-
-.brand-description {
-  font-size: 1.4rem;
-}
-
-.links {
-  display: flex;
-}
-
-.links a {
-  color: #d9d9d9;
-  font-size: 1.2rem;
-  margin: 1rem;
-}
+@import url("https://ozone-ui.netlify.app/index.css");
+
+:root {
+    --white-clr:white;
+    --primary-color:#EF4444;
+    --nav-bg-color :#112457;
+    --prime-blue:#93C5FD;
+    --secondary-pink:#F9ABD4;
+    --info-clr:#A5B4FC;
+    --warning-clr:#FCD34D;
+    --danger-clr:#DC2626;
+    --success-clr:#10B981;
+    --grey-clr:grey;
```

Comment:
you can give this in hex as well just to be consistent

Reply:
sure